q2_chemistree/plugin_setup.py

```diff
@@ -29,17 +29,16 @@
                                         artifact_format=MGFDirFmt)
 
 PARAMS = {
-    'database': Str % Choices(['all', 'PubChem', 'bio']),
+    'database': Str % Choices(['all', 'pubchem']),
     'sirpath': Str,
-    'instrument': Str,
-    'dbcsi': Str,
-    'ppmlim': Int,
+    'instrument': Str % Choices(['qtof', 'orbitrap', 'fticr']),
+    'dbcsi': Str % Choices(['all', 'pubchem', 'bio', 'kegg', 'hmdb']),
+    'ppmlim': Int % Range(0, 30, inclusive_end=True),
     'nproc': Int,
```

Comment:
Can you add a range for the number of processors, and rename the parameter to match other plugins in the qiime2 ecosystem. See for example q2-diversity or q2-feature-classifier.

Reply:
Sure, thanks for the suggestion! Let's use the same parameter names as SIRIUS uses? They are fairly descriptive.

Reply:
That sounds like a good idea granted that they are no redundant or out of context 👍